mrbelvedereci/build/tasks.py

```diff
@@ -98,9 +98,11 @@ def check_queued_build(build_id):
 
     else:
         # For persistent orgs, use the cache to lock the org
-        lock_id = 'mrbelvedereci-org-lock-{}'.format(org.id)
-        status = cache.add(lock_id, 'build-{}'.format(build_id),
-                           timeout=BUILD_TIMEOUT)
+        status = cache.add(
+            org.lock_id,
+            'build-{}'.format(build_id),
```

Comment:
should we have a helper func for the build id? i know its just a simple str format but it seems like we may want to be able to ref that logic again

Reply:
Right, this same string is created elsewhere so it makes sense to consolidate it somewhere. FYI this value is never used (yet).

Reply:
Actually this is the only place it's used, and the value is never read, so I'll leave it be until we have some need for it.